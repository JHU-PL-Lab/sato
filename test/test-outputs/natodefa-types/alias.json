[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "d", "c", "b", "a" ],
          "err_match_val": "input",
          "err_match_expected": "Boolean",
          "err_match_actual": "Integer"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "d and true",
    "err_steps": 216
  }
]