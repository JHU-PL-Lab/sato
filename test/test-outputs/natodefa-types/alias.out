Type errors for:
- Input sequence : [0]
- Found at expr  : d and true
--------------------
* Value    : d = c = b = a = input
* Expected : Boolean
* Actual   : Integer
1 errors found. No further control flows exist.
